py/desisim/templates.py

```diff
@@ -358,7 +358,7 @@ def _blurmatrix(self, vdisp, log=None):
         blurmatrix = dict()
         for uvv in uvdisp:
             sigma = 1.0 + (self.basewave * uvv / LIGHT)
-            blurmatrix[uvv] = pxs.gauss_blur_matrix(self.pixbound, sigma)
+            blurmatrix[uvv] = pxs.gauss_blur_matrix(self.pixbound, sigma).astype('f4')
```

Comment:
I'll create a new issue on the use of double- vs single-precision in the templates code but for now (i.e., for the mocks-to-spectra effort) this part of the code isn't being used.